src/MvvmNano.Forms/MvvmNanoApplication.cs

```diff
@@ -7,6 +8,26 @@ namespace MvvmNano.Forms
     /// </summary>
     public class MvvmNanoApplication : Application
     {
+
+        /// <summary>
+        /// A collection of MasterDetailData that represents available detail pages.
+        /// </summary>
+        public ObservableCollection<MasterDetailData> MasterDetails { get; set; } = new ObservableCollection<MasterDetailData>();
```

Comment:
My critique still applies here. I don't see all this in the main Application. A derived one would be a good idea.